bin/pgenv

```diff
@@ -1424,6 +1449,14 @@ EOF
             cd contrib
             $PGENV_MAKE "${PGENV_MAKE_OPTIONS[@]}"
             $PGENV_MAKE install
+        elif [[ ${v%.*} -ge 17 ]]; then
+            # Starting from 17.x docs target might fail
+            # due to a missing dependency (docbooks-xsl).
+            # Build world without docs, then copy prebuilt
+            # docs from tarball separately.
+            $PGENV_MAKE world-bin "${PGENV_MAKE_OPTIONS[@]}"
+            $PGENV_MAKE install-world-bin
+            cp -R doc/src/sgml/html $PGENV_ROOT/pgsql-$v/share/doc
```

Comment:
Good idea, I've tried. But unfortunately, it won't work. Because the `Makefile` in `doc/src/sgml/` directory responsible for building docs, calls `html` target by default, which that target depends on `html-stamp`, which depends on two generated files `stylesheet.xsl` and `postgres-full.xml` which they get removed somewhere in the process on every build (I might have missed something here and misunderstood the process).
Nontheless, I tried this trick and it didn't work because docs building process starts anyway!